flink-connectors/flink-connector-kinesis/src/test/java/org/apache/flink/streaming/connectors/kinesis/util/KinesisConfigUtilTest.java

```diff
@@ -180,6 +182,104 @@ public void testUnrecognizableCredentialProviderTypeInConfig() {
 		KinesisConfigUtil.validateAwsConfiguration(testConfig);
 	}
 
+	// ----------------------------------------------------------------------
+	// validateEFOConfiguration() tests
+	// ----------------------------------------------------------------------
+
+	@Test
+	public void testNoRecordPublisherTypeInConfig() {
+		Properties testConfig = TestUtils.getStandardProperties();
+
+		KinesisConfigUtil.validateEFOConfiguration(testConfig, new ArrayList<>());
```

Comment:
@xiaolong-sn nit: `Collections.emptyList()` is more expressive

Reply:
I've overload the method, so no need to write the `new ArrayList<>()` now.